logstash/README.md

```diff
@@ -0,0 +1,92 @@
+# Logstash Helm Chart
+
+This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+This helm chart is a lightweight way to configure and run our official [Logstash docker image](https://www.elastic.co/guide/en/logstash/current/docker.html)
+
+## Requirements
+
+* [Helm](https://helm.sh/) >= 2.8.0
+* Kubernetes >= 1.8
+
+## Usage notes and getting started
+
+* This repo includes a number of [example](./examples) configurations which can be used as a reference. They are also used in the automated testing of this chart
+* Automated testing of this chart is currently only run against GKE (Google Kubernetes Engine). If you are using a different Kubernetes provider you will likely need to adjust the `storageClassName` in the `volumeClaimTemplate`
```

Comment:
I know this is copy pasted from the Elasticsearch chart but this isn't true anymore. The chart used to default to literally `default` now its not specified so it will use whatever your actual default storage class is. I think we can just remove this sentence for this chart and Elasticsearch.

Reply:
fixed in https://github.com/elastic/helm-charts/pull/333/commits/ef35da4343010cdcd4112d250009bde97f7f0e8c (for Logstash) and in https://github.com/elastic/helm-charts/pull/335 (for Elasticsearch).